README.md

```diff
@@ -96,7 +96,7 @@ Note: If you're an artist who wants to host a website but doesn't have know how
 
 3. Remove default config file.  
 	```shell
-	rm config.toml
+	rm hugo.toml
```

Comment:
We still should cover `config` for old versions. We need to remove `config` and `hugo` if they exists.